### PR TITLE
Move some pieces outside of src

### PR DESCRIPTION
**Pull request by charleskawczynski**
A step towards #638, and a nice contribution to #628.

### CODE REVIEW

File: driver/dycore.jl [hunk 1]
```diff
@@ -93,3 +93,49 @@ function satadjust(gm::TC.GridMeanVariables, grid, state)
...
+    # Update aux / pre-tendencies filters. TODO: combine these into a function that minimizes traversals
+    # Some of these methods should probably live in `compute_tendencies`, when written, but we'll
+    # treat them as auxiliary variables for now, until we disentangle the tendency computations.
+
+    TC.update_aux!(edmf, gm, grid, state, case, surf, param_set, t, Δt)
+
+    tends_face = tendencies.face
+    tends_cent = tendencies.cent
+    parent(tends_face) .= 0
+    parent(tends_cent) .= 0
```
Comment (jakebolewski) [inline driver/dycore.jl:122]: you no longer have to call `parent` here

Comment (charleskawczynski) [reply driver/dycore.jl:122]: Is this available in ClimaCore 0.6?

Comment (jakebolewski) [reply driver/dycore.jl:122]: hmm looking at this again maybe that didn't make it in yet

Comment (charleskawczynski) [reply driver/dycore.jl:122]: Ok, we can update later

**Review by yairchn — APPROVED** [review]
LGTM results seem unchanged!

**Pull request merged**